kubernetes-model/src/main/resources/schema/kube-schema.json

```diff
@@ -1274,6 +1274,163 @@
         "io.fabric8.kubernetes.api.model.KubernetesResource"
       ]
     },
+    "kubernetes_CustomObject": {
```

Comment:
I assume you added these by hand? Unfortunately, this file is generated by the Go code from Kubernetes source so this will be overwritten each time we generate this file.